src/wp-content/themes/twentyfifteen/js/functions.js

```diff
@@ -5,130 +5,162 @@
  * Contains handlers for navigation and widget area.
  */
 
-( function( $ ) {
-	var $body, $window, $sidebar, resizeTimer,
-		secondary, button;
+( function() {
+	function domReady( callback ) {
```

Comment:
ideally, replace this with our domReady package, which should be available at `wp.domReady` after enqueueing "wp-dom-ready".

Reply:
In a perfect world I'd agree, but it would mean we have to bump the minimum required WordPress version of Twenty Fifteen which is currently 4.1. Given how widely it's probably still used on old sites I'd argue this may not be enough of a benefit to justify a bump.

But for newer themes for sure.

Reply:
Good point!